src/expr/src/scalar/mod.rs

```diff
@@ -77,28 +78,147 @@ pub enum MirScalarExpr {
 }
 
 impl Arbitrary for MirScalarExpr {
-    // TODO: This is just a stub. #11740
     type Parameters = ();
-    type Strategy = Just<MirScalarExpr>;
+    type Strategy = BoxedStrategy<MirScalarExpr>;
```

Comment:
Have you tried if adding `Union<>` fixes the stackoverflow issue? (It's fine to me to merge it without tho)

Reply:
I managed to fix the stack overflow issue by manually implementing `Arbitrary` for `AggregateFunc`.